addons/wowl/static/src/core/download.ts

```diff
@@ -0,0 +1,140 @@
+import download from "../libs/download";
+import { _lt, Stringifiable } from "./localization";
```

Comment:
We talked about the fact that _t was bad in here. _lt seemed to be better, but I still am not sure. If it can't be used, I don't see how we could handle translations in files like this one.